src/components/table/table-cell/table-cell.component.js

```diff
@@ -30,17 +31,26 @@ class TableCell extends React.Component {
   /**
    * Returns if child element is Textarea.
    */
-  childrenHasTextArea() {
+  get checkChildrenInputType() {
     const { children } = this.props;
 
     if (!Array.isArray(children)) {
-      return this.childName(children) === TextArea.name;
+      if (this.childName(children) === TextArea.name) {
+        return 'isTextArea';
```

Comment:
Please could you define these as constants rather than magic strings

Reply:
Yeah I'll add them to the options helper